apps/siiges-app/pages/solicitudes/nuevaSolicitud/index.jsx

```diff
@@ -1,21 +1,10 @@
-import {
-  Card,
-  CardContent,
-  Container,
-  Grid,
-  Step,
-  StepLabel,
-  Stepper,
-  Typography,
-  Box,
-  Divider,
-} from '@mui/material';
-import { ButtonStyled, LoadCircle, Overlay } from '@siiges-ui/shared';
-import React from 'react';
-import ArrowForwardIosIcon from '@mui/icons-material/ArrowForwardIos';
-import { ThemeProvider } from 'styled-components';
-import { InstitucionData } from '@siiges-ui/requests';
-import theme from '../../theme';
+import { Layout } from '@siiges-ui/shared';
+import React, { useState } from 'react';
+import { ModuleHeader, DatosGenerales } from '@siiges-ui/solicitudes';
+import Plantel from '@siiges-ui/solicitudes/src/components/NuevaSolicitud/Plantel';
+import PlanEstudios from '@siiges-ui/solicitudes/src/components/NuevaSolicitud/PlanEstudios';
+import Anexos from '@siiges-ui/solicitudes/src/components/NuevaSolicitud/Anexos';
+import EvaluacionCurricular from '@siiges-ui/solicitudes/src/components/NuevaSolicitud/EvaluacionCurricular';
```

Comment:
This should be a default export, you have to modify how your are exporting things in this packages.  If you make it correct you should be able to export like this
```suggestion
import { dependency } from @siiges-ui/solicitudes
```